src/TRestGeant4PhysicsInfo.cxx

```diff
@@ -0,0 +1,60 @@
+//
+// Created by lobis on 31/03/2022.
+//
+
+#include "TRestGeant4PhysicsInfo.h"
+
+#include <iostream>
+
+using namespace std;
+
+ClassImp(TRestGeant4PhysicsInfo);
+
+void TRestGeant4PhysicsInfo::Print() const {
+    vector<Int_t> ids = {};
+    for (const auto& [id, _] : fProcessNamesMap) {
+        ids.push_back(id);
+    }
+    sort(ids.begin(), ids.end());
+
+    cout << "ProcessNames:" << endl;
+    for (const auto& id : ids) {
+        cout << "\t" << id << " - " << GetProcessName(id) << endl;
+    }
+}
+
+void TRestGeant4PhysicsInfo::InsertProcessName(Int_t id, const TString& processName) {
+    fProcessNamesMap[id] = processName;
+    fProcessNamesReverseMap[processName] = id;
+}
+
+void TRestGeant4PhysicsInfo::InsertParticleName(Int_t id, const TString& particleName) {
+    fParticleNamesMap[id] = particleName;
+    fParticleNamesReverseMap[particleName] = id;
+}
+
+template <typename T, typename U>
+U GetMapValueFromKeyIfExistsOtherwiseGetDefault(const map<T, U>* pMap, const T& key) {
```

Comment:
Not really important, as it's only used internally, but something like `GetOrDefault` is pretty common naming for something like this. Mainly the "if exists otherwise" is pretty much implied.